### PR TITLE
feat(express): support external signer TSS EdDSA in sendMany express endpoint

### PR DESCRIPTION
**Pull request by islamaminBitGo**
Ticket: BG-65857

Test: 

The following transaction was performed using external signer successfully https://explorer.solana.com/tx/gdj8H7jkNZbkAwZeEcNMK1kymmiDAcRR3gXV4Kn1h6DRVTSecUU7gBGXHfpeU4Bh8oHHCjjukkTmANGEfDwuhG4?cluster=devnet

Request:
image

### CODE REVIEW

**Review by jicli — APPROVED** [review]

**Review by olibeyene — CHANGES_REQUESTED** [review]
eddsa already support express endpoint. I don't believe this change is needed. Please confirm

**Comment by islamaminBitGo** [comment]
> eddsa already support express endpoint. I don't believe this change is needed. Please confirm

it's not supported in external signer mode where express would need to call an external signer instance to generate signature shares.

File: modules/express/src/clientRoutes.ts [hunk 1]
```diff
@@ -729,7 +729,11 @@ async function handleV2SendMany(req: express.Request) {
   req.body.reqId = reqId;
   let result;
   try {
-    result = await wallet.sendMany(createSendParams(req));
+    if (wallet._wallet.multisigType === 'tss') {
+      result = await wallet.sendMany(createTSSSendParams(req));
```
Comment (olibeyene) [inline modules/express/src/clientRoutes.ts:733]: can you verify this tss param will work for ECDSA?

Comment (islamaminBitGo) [reply modules/express/src/clientRoutes.ts:733]: we do not support ECDSA yet.

**Review by olibeyene — APPROVED** [review]

**Pull request merged**